front/components/assistant/conversation/SidebarMenu.tsx

```diff
@@ -298,13 +298,10 @@ export function AssistantSidebarMenu({ owner }: AssistantSidebarMenuProps) {
                                 isMultiSelect
                                   ? () => (
                                       <Checkbox
-                                        variant="checkable"
-                                        checked={
-                                          selectedConversations.includes(c)
-                                            ? "checked"
-                                            : "unchecked"
-                                        }
-                                        onChange={() =>
+                                        defaultChecked={selectedConversations.includes(
```

Comment:
looking at onCheckedChange it seems the state of the checkbox can be kept here, why not controlling checkbox by using `checked` instead of `defaultChecked` ?

Reply:
Seen IRL, got rid of "defaultChecked" https://github.com/dust-tt/dust/pull/8001